documentation/spring-integration/index.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: docs
+title: Documentation — Spring integration
+submenu: spring-integration
+---
+
+# Spring integration
+
+<h2 name="introduction">Introduction</h2>
+
+One of our main approach is Javers should be easy to use, with this in mind we prepared integration for
```

Comment:
brakuje tu kontekstu, po co mam dodawać te adnotacje, co mi to da? I gdzie mam je dawać?

ze zdania 'annotate a methods that modify your data' można wywnioskować że np na metodach Encji. Ani słowa o repozytoriach i o tym, że annotacje wykonują przez aspekt commity do JaversRepository, co zapewnia automatyczy audyt danych ...